pages/[...path_params].jsx

```diff
@@ -76,9 +76,12 @@ export default function Page(props) {
     // The first element is an empty string so take it out
     breadcrumbs.shift();
 
-    return `Drash Land - ${topBarModuleName} - ${
-      formatLabel(titleCase(breadcrumbs[breadcrumbs.length - 1]))
-    }`;
+    let title = formatLabel(titleCase(breadcrumbs[breadcrumbs.length - 1]))
+    title = title
+      .replace("Tutorials Node", "Tutorials (Node)")
+      .replace("Tutorials Deno", "Tutorials (Deno)");
```

Comment:
```suggestion
    consttitle = formatLabel(titleCase(breadcrumbs[breadcrumbs.length - 1]))
      .replace("Tutorials Node", "Tutorials (Node)")
      .replace("Tutorials Deno", "Tutorials (Deno)");
```

Reply:
i tried doing this, but i was running into issues with the code blocks overlapping (e.g., you click on Node - TypeScript for Mocks, navigate to Node - TypeScript for Spies, and it would show Node - TypeScript for Mocks).

imma make another issue to combine the Deno and Node docs. also, now that Bun is in the JS/TS picture, it would be best to only have example code blocks with tabs as opposed to duplicated code blocks, so i do agree with you that it would make things much easier if we used tabs instead

Reply:
made the issue: https://github.com/drashland/website-v2/issues/185